app/views/users/show.html.erb

```diff
@@ -1,21 +1,37 @@
-<div class="user">
-  <%= render partial: 'user', locals: { user: @user } %> 
-  <div class="user-bio">
-    <h3>Bio</h3>
-    <%= @user.bio %>
+<section class='main_section'>
+  <div class='user_card'>
+    <div class='user_img'>
+      <img src= <%= @user.photo %> alt= <%= @user.name %> />
+    </div>
+
+    <div class='user_stuff'>
+      <p class='user_name'><%= @user.name %></h2>
+      <p class='posts_no'>Number of Posts: <%= @user.postscounter || 0 %></p>
```

Comment:
- [ ] It will be great to ensure your app runs without any errors. After fixing the error in the user index page, there is another error here. Kindly check below ⬇️ 
![Screenshot 2023-05-27 at 14 38 12](https://github.com/LeslieAine/blog-app1/assets/32361963/507ce496-8e75-4661-a022-bb604b5abfa8)

This error is caused by the fact that you are calling `@user.postscounter ` but the user doesn't have a `postscounte`r, only the post has that. I tried fixing it but it was taking too long. Kindly ensure your application runs smoothly without any errors.

Reply:
Thank you @peterrobert for reviewing my code and leaving suggestions 🙏🏽